### PR TITLE
Add datomic docs #160

### PR DESCRIPTION
**Pull request by mtsbarbosa**
#160
Add datomic docs

Recreated the PR after first one got deleted when I deleted my fork by mistake

### CODE REVIEW

File: content/guides/datomic/project.clj [hunk 1]
```diff
@@ -0,0 +1,11 @@
+(defproject hello "0.1.0-SNAPSHOT"
+            :description "Hello World with pedestal and datomic!"
+            :repositories [["my.datomic.com" {:url "https://my.datomic.com/repo"
+                                              :creds :gpg}]] ;;<1>
+            :dependencies [[org.clojure/clojure "1.11.1"]
+                           [io.pedestal/pedestal.service "0.5.7"]
+                           [io.pedestal/pedestal.route "0.5.7"]
+                           [io.pedestal/pedestal.jetty "0.5.7"]
+                           [org.slf4j/slf4j-simple "1.7.28"]]
+            :resource-paths ["config", "resources"]
```
Comment (hlship) [inline content/guides/datomic/project.clj:10]: Dependency on datomic-pro appears missing here.

File: content/guides/datomic/src/ports.datomic/peer.clj [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+(def uri "datomic:dev://localhost:4334/hello")
+
+(def schema-tx (read-string (slurp "resources/hello/schema.edn")))
+(def data-tx (read-string (slurp "resources/hello/seed-data.edn")))
+
+(defn init-db []
+      (println "starting db!")
+      (d/create-database uri)
+      (let [conn (d/connect uri)]
+           (println @(d/transact conn schema-tx))
```
Comment (hlship) [inline content/guides/datomic/src/ports.datomic/peer.clj:13]: I'd prefer to see this as structured logging.

File: content/guides/pedestal-with-datomic.adoc [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+== Guide Assumptions
+This tutorial extends the link:hello-world[Hello World Service] to use strings retrieved
+from Datomic. We will start from the "hello" service that we created
+in our first tutorial.
+
+This tutorial assumes that you have Datomic downloaded already.
+If not, hop over to the https://www.datomic.com/get-datomic.html[download site].
+
+== Add Datomic to the Project
+After downloading the datomic
```
Comment (hlship) [inline content/guides/pedestal-with-datomic.adoc:29]: this looks truncated, should be something like "After downloading and installing the Datomic distibution ..."

File: content/guides/pedestal-with-datomic.adoc [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+Add credentials to lein folder:
+----
+$ touch ~/.lein/credentials.clj.gpg
```
Comment (hlship) [inline content/guides/pedestal-with-datomic.adoc:32]: Why `touch` why not say something like "follow the directions here to set up your credentials and encrypt them."

File: content/guides/pedestal-with-datomic.adoc [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+- Test your service using Pedestal's test helpers.
+
+== Guide Assumptions
+This tutorial extends the link:hello-world[Hello World Service] to use strings retrieved
+from Datomic. We will start from the "hello" service that we created
+in our first tutorial.
+
+This tutorial assumes that you have Datomic downloaded already.
+If not, hop over to the https://www.datomic.com/get-datomic.html[download site].
+
```
Comment (hlship) [inline content/guides/pedestal-with-datomic.adoc:27]: Maybe some more intro: "You will run a development mode Datomic instance locally in one process, and run a Pedestal-based server in another."

File: content/guides/datomic/src/ports.datomic/peer.clj (hunk 2 above)
Comment (mtsbarbosa) [reply content/guides/datomic/src/ports.datomic/peer.clj]: I added pedestal log into it, not sure if it suits well the use case here

**Pull request closed without merging**